src/event_loop.rs

```diff
@@ -224,14 +224,26 @@ impl<T> EventLoop<T> {
     ///   (that Rust doesn't see) that will also mean that the rest of the function is never executed
     ///   and any values not passed to this function will *not* be dropped.
     ///
-    ///   Web applications are recommended to use `spawn()` instead of `run()` to avoid the need
+    ///   Web applications are recommended to use
+    #[cfg_attr(
+        wasm_platform,
+        doc = "[`EventLoopExtWebSys::spawn()`][crate::platform::web::EventLoopExtWebSys::spawn()]"
```

Comment:
```suggestion
        doc = "[`EventLoopExtWebSys::spawn()`](crate::platform::web::EventLoopExtWebSys::spawn())"
```
Is the convention we are using as far as I'm aware.

Reply:
I'm fairly confident that `rustdoc` defines `[foo][bar]` as extension to link to symbols (in addition to a label name): https://rust-lang.github.io/rfcs/1946-intra-rustdoc-links.html#additions-to-the-documentation-syntax

Hence there must either be a label definition (e.g. `/// [bar]: ...`) _or_ `bar` must be a valid Rust path in scope.

On the contrary, `[foo](bar)` is defined as `bar` being a URL, so there are cases where an invalid Rust path is (were?) treated as link, resulting in no errors at build-time and a useless/broken link for the user.

---

I am referencing to the past here because I **do** now get build-time errors for these, but am still wary of edge-cases. And while `[]()` has precedence, `[][]` is definitely used in `winit` as well.

Reply:
> The point is that it supports that _in addition to URLs_.

Absolutely, I understand, it's why this totally makes sense to me!
Thank you for the explanation!

Reply:
Little spark; found it: https://github.com/gtk-rs/gir/pull/1162

> Rustdoc treats `[foo]` as item path, and by the same rule `[fancy text][foo]` too. Unlike `[]()`, which generates a dead link if the right-hand side is a valid Rust item path but doesn't exist in scope (common and unfortunately unavoidable with docs for C code auto-ported to Rust), the `[]` and `[][]` do not generate a link at all.

And there are more details in the whole chain of linked issues and PRs:

https://github.com/gtk-rs/gir/issues/1140#issuecomment-842131324
https://github.com/gtk-rs/gir/pull/1150

Reply:
Fair enough, I think I replaced those locally with the symbol rather than going through a named link so had a few more, but abandoned that change for the sake of keeping this PR "small".